### PR TITLE
add cluster limit

### LINKED ISSUE #2581: Document: feat: introduce cluster limit
This issue tracks the documentation update needed for the merged PR #18383.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/18383
Source PR Merged At: 2024-09-06T07:16:37Z

If it is a major improvement that deserves a new page or a new section in the documentation, please check if we should label it as an experiment feature.

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Related code PR

https://github.com/risingwavelabs/risingwave/pull/18383

## Related doc issue

Resolve https://github.com/risingwavelabs/risingwave-docs/issues/2581

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/deploy/cluster-limit.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  - Increase the limit via meta developer config:
+
+  ```toml
+  [meta.developer]
+  meta_actor_cnt_per_worker_parallelism_soft_limit = 100
+  meta_actor_cnt_per_worker_parallelism_hard_limit = 400
+  ```
+
+  However, please keep in mind, if you bypass the check or increase the limits, the cluster could become overloaded, leading to problems with stability, availability, or performance. Please be careful and proceed at your own risk.
```
Comment (hzxa21) [inline docs/deploy/cluster-limit.md]: Can we put this in a highlighted block? I would like to warn user about the risk of bypassing the check.

**Review by hzxa21 — APPROVED** [review]
Rest LGTM

File: docs/deploy/cluster-limit.md (hunk 1 above)
Comment (WanYixian) [reply docs/deploy/cluster-limit.md]: Just split "override limit" part from notes, and highlight the caution
image

**Comment by WanYixian** [comment]
@hzxa21 Here's the latest [preview.](https://pr-2600.d2fbku9n2b6wde.amplifyapp.com/docs/next/cluster-limit/) Please take another look. Will merge this PR if no questions.

**Review by hzxa21 — APPROVED** [review]
Thanks for the efforts!

**Pull request merged**